assignments/prerequisites/python-programming-introduction.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>Merge this cell with above cell. Instead of using <code>print</code>,please use <code>assert</code> to check all the nodes in the merged_list.</p><p> </p><p>You can address this in your future PR. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/349/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>